### DISCUSSION TITLE
Rule Proposal: Sort the standalone imports

### DISCUSSION THREAD

**Discussion by alexandrevryghem** [angular-eslint/angular-eslint#2179] [New Rule Proposals]
In order to make it easier to resolve merge conflicts I made [a custom linting rule](https://github.com/alexandrevryghem/dspace-angular/commit/b9157c2422ef2bc39c7afdd674c0e2f52c4682bd) that enforces standalone imports to be sorted and written on a separate line. I was wondering if it was something that was maybe worth contributing here as well

**Invalid format:**
The imports are written in a compact, inline style, making changes harder to track in diffs:
```ts
@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.scss'],
  standalone: true,
  imports: [RootComponent, AsyncPipe],
})
```
**Valid format:**
Each import is placed on its own line for better readability:
```ts
@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.scss'],
  standalone: true,
  imports: [
    AsyncPipe,
    RootComponent,
  ],
})
```

**Comment by reduckted** [comment]
I like the reasoning (avoiding merge conflicts), but there used to be a rule to sort imports in `NgModule`, but it was deprecated, so this new rule is unlikely to be added.

#1781

**Reply by JamesHenry** [reply]
This is correct, thanks @reduckted
